funcframework/framework.go

```diff
@@ -32,10 +32,11 @@ import (
 )
 
 const (
-	functionStatusHeader = "X-Google-Status"
-	crashStatus          = "crash"
-	errorStatus          = "error"
-	panicMessageTmpl     = "A panic occurred during %s. Please see logs for more details."
+	functionStatusHeader     = "X-Google-Status"
+	crashStatus              = "crash"
+	errorStatus              = "error"
+	panicMessageTmpl         = "A panic occurred during %s. Please see logs for more details."
+	fnErrorMessageStderrTmpl = "Function error: %v"
```

Comment:
It only applies to when the function returns an error, which is only possible with CloudEvent and Background Functions. The template is used in both of those cases.

Reply:
This error is what users see today when Background Functions return an error. This simply turns it into a template so that CloudEvent Functions use the same message when returning an error. It is prefixed with "Function error" to indicate the user function is returning an error. It is not intended to be a generic any-functions-framework prefix.